plugins/destination/sqlite/docs/overview.md

```diff
@@ -3,21 +3,11 @@ name: SQLite
 stage: GA
 title: SQLite Destination Plugin
 description: CloudQuery SQLite destination plugin documentation
+hub-order: 1
 ---
+
 # SQLite Destination Plugin
```

Comment:
```suggestion
```

We can remove this I think as now we have duplicated headers 
<img width="858" height="371" alt="image" src="https://github.com/user-attachments/assets/4f6e883f-ce42-4cd4-af64-11c7faa63faf" />